### ISSUE TITLE
Draft mode broken

### ISSUE THREAD

**Issue by dmiddleton** [dragonfly-science/mpi-latex-templates#22]
In the current version (1.24, 20 Sept 2019) draft mode seems to be broken.

On pisces, using \documentclass[draft]{mpi-far} gives the following error:

```
! Undefined control sequence.
\@endpreamblehook ...terPreamble \@firstofone  \g 
                                                  @addto@macro\maketitle { \...
l.95 \begin{document}
```

**Comment by edwardabraham** [comment]
Can confirm that I have the same issue.  I spent ages trying to figure out why my report wasn't working. Sigh.

**Comment by edwardabraham** [comment]
Ok. I have fixed this! The updated debian version is 1.25
